packages/logger/tests/unit/configFromEnv.test.ts

```diff
@@ -0,0 +1,198 @@
+/**
+ * Test Logger EnvironmentVariablesService class
+ *
+ * @group unit/logger/config
+ */
+import { beforeEach, describe, expect, it, jest } from '@jest/globals';
+import { EnvironmentVariablesService } from '../../src/config/EnvironmentVariablesService.js';
+
+describe('Class: EnvironmentVariablesService', () => {
+  const ENVIRONMENT_VARIABLES = process.env;
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env = { ...ENVIRONMENT_VARIABLES };
+  });
+
+  it('returns the value of the environment variable AWS_LAMBDA_LOG_LEVEL and aliases it as needed', () => {
+    // Prepare
+    process.env.AWS_LAMBDA_LOG_LEVEL = 'FATAL';
+    const service = new EnvironmentVariablesService();
+
+    // Act
+    const value = service.getAwsLogLevel();
+
+    // Assess
+    expect(value).toEqual('CRITICAL');
```

Comment:
The only thing that seems odd here is that you SET a `FATAL` level, but compare it to `CRITICAL` and it's true. I know why it's true, but if you add comments in Prepare/Act/Assess, it might help others to understand this test.

Reply:
Good point, will add a comment to explain the quirk.